Documentation/ApiOverview/LinkBrowser/Tutorials/CustomLinkBrowser.rst

```diff
@@ -0,0 +1,264 @@
+    Can be used to decide the order of the tabs.
+
+:typoscript:`scanAfter` / :typoscript:`scanBefore`
+    The first backend link handler who determines that it can :ref:`handle the link
+    <tutorial_backend_link_handler_canHandleLink>` may edit a link. The
+    external url accepts any link no other link handler accepts. Therefore
+    we must scan before it.
+
+:typoscript:`configuration`
+    Some configuration, available to the backend link handler. This information
+    is **not available in the frontend**. Therefore in the :ref:`frontend
+    rendering of the link <tutorial-typolink-builder>` the information must be
+    stored in another way. In this example we hardcoded it. But you could also
+    make it available by TypoScript Setup or as part of the link that is saved.
+
+.. _tutorial_backend_link_handler:
+
+2. Create a link browser tab
+============================
+
+To create a link browser tab we implement the interface
+:php:`\TYPO3\CMS\Backend\LinkHandler\LinkHandlerInterface`.
+
+All backend link handlers provided by the Core extend the abstract class
+:php:`TYPO3\CMS\Backend\LinkHandler\AbstractLinkHandler`. However, this class is
+marked as :php:`@internal` and therefore can be changed by the Core Team at any time.
+
+You have the choice of implementing the :php:`LinkHandlerInterface` yourself by
+having a look at the :php:`AbstractLinkHandler` for best practices or to extend
+the :php:`AbstractLinkHandler`. In the latter case your code might break on
+updates though.
+
+In this tutorial, we implement the :php:`LinkHandlerInterface` directly, as it is
+best practice not to rely on internal classes.
+
+You can find the complete class in the extension EXT:examples on GitHub:
+`GitHubLinkHandler <https://github.com/TYPO3-Documentation/t3docs-examples/blob/main/Classes/LinkHandler/GitHubLinkHandler.php>`__.
+
+We will explain some of the important methods below:
+
+Initialization and dependencies
+-------------------------------
+
+..  include:: /ApiOverview/LinkBrowser/Tutorials/_CustomLinkBrowser/_GitHubLinkHandlerInitialize.rst.txt
+
+For technical reasons, not all dependencies needed by the backend link handler can
+be acquired by :ref:`DependencyInjection`. Therefore the following two methods
+are called by Core classes once the dependencies are available:
+
+:php:`LinkHandlerInterface::initialize()` takes care of setting the
+:php:`\TYPO3\CMS\Backend\Controller\AbstractLinkBrowserController`, the identifier and
+the configuration information. In this example we only need the configuration,
+the other parameters might be needed in different scenarios.
+
+:php:`AbstractLinkBrowserController $linkBrowser`
+    Is the surrounding class calling the link handler. This class stores
+    configuration information on the complete link browser window.
+
+:php:`string $identifier`
+    Contains the key of the page TSconfig configuration of the link browser tab
+    this instance renders.
+
+:php:`string[] $configuration`
```

Comment:
I suspect, string[] is not correct. Valid page TSconfig may be:

[
    'some_key' => 'some_value'
]

which is equivalent to `array<string, string>`, but also `array<string, array<string, string>>` is valid.

But string[] means:

[
    'some_value'
]